README.md

```diff
@@ -2,14 +2,45 @@
 
 This repository was developed to support the *Esri Dev Summit 2018* presentation of similar name.  In the [samples](https://github.com/sean-olson-esri/2018_dev_summit_arcgis_api_angular_cli/tree/master/samples) directory you will find a number of implementations of the ArcGIS API for JavaScript in Angular (2+) applications, using the [`esri-loader`](https://github.com/Esri/esri-loader).
 
-
 ### In this Repository
+
+This repository contains multiple, standalone Angular application samples that share a top-level `.angular-cli.json`* and `package.json`. This saves us from having many individual Angular CLI configurations and just as many repeats of `node_modules` dependencies. The catch here is that we will have more verbose `ng ...` commands to ensure that we are pointing the Angular CLI to an appropriate sample application directory.
+
+*not to sidetrack you, but if you're curious: https://github.com/angular/angular-cli/wiki/stories-multiple-apps
+
+#### Getting started
+
+_TODO: update this section with info on how we got here in the first place? i.e. global angular cli, esri-loader, esri types, angular devkit?_
```

Comment:
Yep, maybe a short paragraph explaining how this repo pulls all the bits together.